README.md

```diff
@@ -126,6 +126,18 @@ Note that,
   * On ARM
     * Neon
 
+### Building Velox with docker-compose
+
+If you don't want to install the system dependencies required to build Velox,
+you can also build and run tests for Velox on a docker container
+using [docker-compose](https://docs.docker.com/compose/).
+Use the following commands:
+
+```shell
+$ docker-compose build ubuntu-cpp
```

Comment:
Would changes to our scripts break docker-compose ? Should we have a ci job to ensure this doesnt break too ?

Reply:
if the script `setup-ubuntu.sh` works on ubuntu 22.04 which is the base for the docker image they should work on the docker-compose image. Is the `setup-ubuntu.sh` script tested on CI today? Otherwise we could validate it adding a new CI job that builds and runs this image.

Reply:
AFAIK , we dont run setup-ubuntu on any regular cadence for Velox . Its run when creating the docker images, but not after that on Velox. In TorchArrow however it is run regularly; but theres a delay from when changes are made and when the velox submodule is updated. We should probably have CI job for this until we are able to migrate our dependencies to CMake.

Reply:
We can do that on another PR though.